invenio/modules/checker/rules.py

```diff
@@ -0,0 +1,293 @@
+# -*- coding: utf-8 -*-
+##
+## This file is part of Invenio.
+## Copyright (C) 2013, 2014 CERN.
+##
+## Invenio is free software; you can redistribute it and/or
+## modify it under the terms of the GNU General Public License as
+## published by the Free Software Foundation; either version 2 of the
+## License, or (at your option) any later version.
+##
+## Invenio is distributed in the hope that it will be useful, but
+## WITHOUT ANY WARRANTY; without even the implied warranty of
+## MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+## General Public License for more details.
+##
+## You should have received a copy of the GNU General Public License
+## along with Invenio; if not, write to the Free Software Foundation, Inc.,
+## 59 Temple Place, Suite 330, Boston, MA 02111-1307, USA.
+
+"""Rule handlers for checker module."""
+
+import json
+import yaml
+from argparse import ArgumentTypeError
+from collections import defaultdict, MutableSequence
+from importlib import import_module
+from intbitset import intbitset
+from pykwalify.core import Core  # TODO: Remove when cfg is moved to db
```

Comment:
This configuration is moving to the database as per the comment. It will be then converted to a dict to interface with modules like this, at which point we can use cerberus, although I believe there should be no need for that if the converted is thoroughly tested.